### DISCUSSION TITLE
update registration

### DISCUSSION THREAD

**Discussion by GregSchiemer** [iterate-ch/cyberduck#16122] [Q&A] [answered]
I’ve been very happy using the free version of Cyberduck [ Version 867003868] and just purchased the paid version on Apple Store. But it still shows as Unregistered. How do I update that ?

**Comment by dkocher** [comment, marked as answer]
You will need to replace your existing installation by trashing Cyberduck.app and reinstalling from the Mac App Store. Opening the application installed from the Mac App Store once will also copy the receipt to the application support folder which will allow the application downloaded from https://cyberduck.io show as registered if you may decide to switch to this version again at a later point.

**Comment by GregSchiemer** [comment]
That fixed it, not really a problem. Thanks for a great product.
